src/styles/header.module.css

```diff
@@ -15,8 +20,21 @@
   margin-left: 100px;
 }
 
+.logo {
+  font-family: 'Montserrat', Arial, Helvetica, sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  color: #0290ff;
+}
+
+.profile img {
+  height: 2.813rem;
+}
+
 .navbarItem > * {
```

Comment:
- [ ] Great job so far @Estete9 There are a few changes I will suggest you implement and you'll be on your way to the next milestone 😉 
- [ ] Your `Navbar` currently has a background color different from that of the given Zeplin design so kindly fix that 👍
- [ ] You can also see that your font size for `Books and Categories` looks a little smaller and there are no spaces between the letters as seen in the Zeplin design so kindly fix that as well 😉 
- Your Design
![image](https://github.com/Estete9/bookstore-react/assets/102699958/734319f4-2120-4303-bc0b-283e9a38579b)

- The Zeplin Design
![image](https://github.com/Estete9/bookstore-react/assets/102699958/21d363d4-4671-4425-b8fd-7de1e60abda2)